Formula/testkube.rb

```diff
@@ -0,0 +1,36 @@
+class Testkube < Formula
+  desc "☸️ Kubernetes-native framework for test definition and execution"
+  homepage "https://testkube.io"
+  url "https://github.com/kubeshop/testkube/archive/refs/tags/v0.10.1.tar.gz"
+  sha256 "ddcc802c4150bf456dcc3c4b0a3323675ad41e4b200908d4eec23aadb27063d1"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    user = "brew"
+    tracking_id = "UA-204665550-8"
+    ENV["CGO_ENABLED"] = "0"
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.builtBy=#{user}
+      -X github.com/kubeshop/testkube/pkg/analytics.testkubeTrackingID=#{tracking_id}
+    ]
+    (buildpath/"src/github.com/kubeshop/testkube").install buildpath.children
+    ENV.append_path "PATH", buildpath/"bin"
+
+    cd "src/github.com/kubeshop/testkube" do
+      system "go", "build", *std_go_args(output: "kubectl-testkube", ldflags: ldflags), "cmd/kubectl-testkube/main.go"
+      bin.install "kubectl-testkube"
```

Comment:
This is already done by `output:`

Reply:
Without it we got
```Running `brew update --preinstall`...
==> Downloading https://github.com/kubeshop/testkube/archive/refs/tags/v0.10.1.tar.gz
Already downloaded: /home/vsukhin/.cache/Homebrew/downloads/fef92fb8fce69ca59dd35e6e81aac0090f318a85099e850a9f670d3517800884--testkube-0.10.1.tar.gz
==> go build -o=kubectl-testkube -ldflags=-s -w -X main.version=0.10.1 -X main.builtBy=Homebrew -X github.com/kubeshop/testkube/pkg/analytics.testkubeTrackingID
Error: Empty installation
```

Reply:
Ah right, output should be `bin/"kubectl-testkube"`